node/src/chain_spec.rs

```diff
@@ -132,7 +136,17 @@ pub fn testnet_config() -> ChainSpec {
 fn testnet_genesis(
     sudo_key: AccountId,
     mut endowed_accounts: Vec<AccountId>,
-    initial_authorities: Vec<(AccountId, AccountId, BabeId, GrandpaId, ImOnlineId)>,
+    initial_authorities: Vec<(
```

Comment:
In stagenet branch these keys merged into single structure for easy using as positional arguments are kind of mess.